rest-api/tools/import_organizations.py

```diff
@@ -204,6 +275,38 @@ def run(self, filename, dry_run):
             logging.info('Starting import of test participants.')
             self._insert_new_participants(self.new_sites_list)
 
+
+  def delete_sql_statement(self, session, site_id_list):
+    str_list = ','.join([str(i) for i in site_id_list])
+    sql = """
+          DELETE FROM site
+          WHERE site_id IN ({str_list}) 
+          AND NOT EXISTS(
+          SELECT * FROM participant p, participant_history ph
```

Comment:
See above re: NOT_EXISTS. Also, you should check site ID columns on participant_summary, physical_measurements, biobank_order here.

Reply:
done. and there is no site_id on PM and bio tables